### DISCUSSION TITLE
Can't deploy the vineyard cluster on the kubernetes cluster created by arm machine

### DISCUSSION THREAD

**Discussion by giscus** [v6d-io/v6d#1457] [General]
# notes/cloud-native/deploy-kubernetes

Vineyard is managed by the Vineyard Operator on Kubernetes. Install vineyard-operator: There are two recommended methods for installing the vineyard operator: using Helm (preferred) or installing d...

https://v6d.io/notes/cloud-native/deploy-kubernetes.html

**Comment by jingchu000** [comment]
kubectl  logs -n vineyard-system      vineyardd-sample-etcd-0      -p                                                       
etcd on unsupported platform without ETCD_UNSUPPORTED_ARCH=arm64 set

**Comment by dashanji** [comment]
Hi @jingchu000, are you using a kubernetes cluster based on the ARM64? At present, we only support the amd64.

**Reply by jingchu000** [reply]
I'm using kind to build the cluster.

**Reply by dashanji** [reply]
Thanks the report, I opened an issue to address the problem. FYI, https://github.com/v6d-io/v6d/issues/1458.

**Comment by dashanji** [comment]
Hi @jingchu000, we have fixed the bug via #1459. You could download the latest vineyard image and it should work on your arm machine.
